web/package.json

```diff
@@ -16,6 +16,8 @@
     "raf": "3.4.0"
   },
   "scripts": {
+    "precommit": "lint-staged",
+    "lint": "eslint src",
     "start": "REACT_APP_BACKEND_SERVICE=http://192.168.19.96:30020/api/v1 node scripts/start.js",
```

Comment:
Hard code `192.168.19.96:30020`?

Reply:
it's for test, can be removed when the api-proxy is ready